Source/Core/Core/HW/DVD/DVDInterface.cpp

```diff
@@ -1189,24 +1192,24 @@ void ExecuteCommand(ReplyType reply_type)
   {
     if (s_DICMDBUF[0] == 0xFF014D41 && s_DICMDBUF[1] == 0x54534849 && s_DICMDBUF[2] == 0x54410200)
     {
-      INFO_LOG(DVDINTERFACE, "Unlock test 1 passed");
+      INFO_LOG_FMT(DVDINTERFACE, "Unlock test 1 passed");
     }
     else if (s_DICMDBUF[0] == 0xFF004456 && s_DICMDBUF[1] == 0x442D4741 &&
              s_DICMDBUF[2] == 0x4D450300)
     {
-      INFO_LOG(DVDINTERFACE, "Unlock test 2 passed");
+      INFO_LOG_FMT(DVDINTERFACE, "Unlock test 2 passed");
     }
     else
     {
-      INFO_LOG(DVDINTERFACE, "Unlock test failed");
+      INFO_LOG_FMT(DVDINTERFACE, "Unlock test failed");
     }
   }
   break;
 
   default:
-    ERROR_LOG(DVDINTERFACE, "Unknown command 0x%08x (Buffer 0x%08x, 0x%x)", s_DICMDBUF[0], s_DIMAR,
-              s_DILENGTH);
-    PanicAlertT("Unknown DVD command %08x - fatal error", s_DICMDBUF[0]);
+    ERROR_LOG_FMT(DVDINTERFACE, "Unknown command {:#08x} (Buffer {:#010x}, {:#x})", s_DICMDBUF[0],
```

Comment:
Shouldn't the `{:#08x}` here be `{:#010x}`instead?

Reply:
Yep. Good catch!